PactNet/Mocks/MockHttpService/AdminHttpClient.cs

```diff
@@ -76,6 +78,11 @@ public async Task SendAdminHttpRequest(HttpVerb method, string path)
             Dispose(request);
             Dispose(response);
 
+            if (path == "/pact" && headers != null && headers.ContainsKey("fileNameAndPath"))

```

Comment:
I think this approach my not be the best one for a couple of reasons:

1. Using headers here look like workaround. You pass file path as a header and store pact response content, but is not need to be pass to mock server. Furthermore mock server will ignore it. 
2. Why you decide to store pact file in this class at first place? Is just simple http client class, which IMHO should be responsible for handling http request/response. I can only guess that it is because this method do not return response content. However, won't it be more sensible to return http response or response content from http client instead of affecting it with some pact specific logic?
3. Pact cli, used for stand alone mock server can be run with `--pact-dir`, this is how current implementation works so it may looks a bit less consistent. 
4. You have duplicated `/pact` uri path

From my point of view you have two of option to consider:

1. Change pact-net client in a way it will always be responsible for storing pacts in both cases (attached or stand alone ). Consider mock provider or builder class to be responsible for storing instead of http client. As I have mention above  you will need to return at least http content from http client class.
2. Make stand alone mock server responsible for storing pact file as it is now, However this will probably need to figure out what to do with few PactDir property in `PactBuilder` class.

Reply:
Looking into this :)

Reply:
That a good idea! Much cleaner, I have adjusted the code accordingly, thank you!